config/rollup/plugin-styles.js

```diff
@@ -25,6 +28,7 @@ module.exports.styles = function styles({
   const renderSass = promisify(nodeSass.render);
 
   const styleProcessor = postcss([
+    pxtorem(pixelsToRemsConfig),
```

Comment:
This should live in the same place that we configure `@shopify/postcss-plugin`, rather than as part of the rollup plugin - in the `plugins` argument that is passed into this plugin - https://github.com/Shopify/polaris-react/blob/main/loom.config.ts#L143 (and shortly below).

If we do that then you can have a config file that exports an array of postcss plugins that are used, and then you can use that array in both the loom config where postcss plugins are defined, and in storybook config

```js
// config/postcss-plugins.js
const postcssShopify= require('@shopify/postcss-plugin');
const pxtorem = require('postcss-pxtorem');

module.exports = [
  postcssShopify,
  pxtorem({
    rootValue: 10,
    replace: true,
    propList: ['*'],
    selectorBlackList: [],
  })
]
```